File: matlab/FizzBuzz.m
for i=1:100
    output = "";
    if mod(i, 3) == 0
        output = output + "Fizz";
    end
    if mod(i, 5) == 0
        output = output + "Buzz";
    end
    if output == ""
        output = int2str(i);
    end
    fprintf("%s\n", output)
end

File: go/fizzbuzz.go
package main

import "fmt"

func main() {
	for i := 1; i <= 100; i++ {
		s := ""
		if i%3 == 0 {
			s += "Fizz"
			if i%5 == 0 {
				s += "Buzz"
			}
		} else if i%5 == 0 {
			s += "Buzz"
		}
		if len(s) > 0 {
			fmt.Println(s)
		}
	}
}
